Source/WebCore/css/mathml.css

```
@@ -155,6 +155,12 @@ ms, mtext, mi, mn, mo, annotation, mtd {
     white-space: nowrap !important;
 }
 
+/* Fractions */
+mfrac {
+    padding-inline-start: 1px;
+    padding-inline-end: 1px;
```

```suggestion
    padding-inline: 1px;
```

This can be shortened

---

Done in 3f75cc7.